infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -0,0 +1,27 @@
+const taskcluster = require('taskcluster-client');
+const assert = require('assert');
+
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Create role and expand smoketest (--target roles)',
+  requires: [],
+  provides: [
+    'target-roles',
+  ],
+  run: async () => {
+    let auth = new taskcluster.Auth(taskcluster.fromEnvVars());
+    let randomId = taskcluster.slugid();
+    let roleId = `project:taskcluster:smoketest:${randomId}`;
+    let payload = {
+      description: 'smoketest test',
+      scopes: ['project:taskcluster:smoketest:*'],
+    };
+    let expandPayload = {
+      scopes: [`assume:${roleId}`],
+    };
+    let roleCreated = await auth.createRole(roleId, payload);
+    let expandedRole = await auth.expandScopes(expandPayload);
+    assert.deepEqual(roleCreated.expandedScopes, expandedRole.scopes, "Error");
```

Comment:
When this fails, it looks like this:
```
AssertionError [ERR_ASSERTION]: Error
```

which isn't so useful.  If you don't give `assert` its last argument, it creates a useful message:

```
✖ Create role and expand smoketest (--target roles) (AssertionError [ERR_ASSERTION]: [ 'assume:project:taskcluster:smoketest:a9HU5LivSQCdZ6RHC9WgjA',
   'project:taskcluster:smoketest:*' ] deepEqual [ 'assume:project:taskcluster:smoketest:a9HU5LivSQCdZ6RHC9WgjA',
   'project:taskcluster:smoketest:*',
   'foo' ])
```

```suggestion
    assert.deepEqual(roleCreated.expandedScopes, expandedRole.scopes);
```

Reply:
Oh, I don't have permission to merge this change -- if you can do it, then @mention me, I'll merge the PR.

Reply:
Yes. I have made the changes. can you merge it please.
@djmitche